Tests/test_Entrez_online.py

```diff
@@ -129,11 +129,12 @@ def test_seqio_from_url(self):
         """Test Entrez into SeqIO.read from URL."""
         handle = Entrez.efetch(db="nucleotide", id="186972394", rettype="gb",
                                retmode="text")
-        self.assertTrue(handle.url.startswith(URL_HEAD + "efetch.fcgi?"), handle.url)
-        self.assertIn(URL_TOOL, handle.url)
-        self.assertIn(URL_EMAIL, handle.url)
-        self.assertIn(URL_API_KEY, handle.url)
-        self.assertIn("id=186972394", handle.url)
+        url = handle.buffer.url
```

Comment:
This is a minor frustration at one level as using ``handle.url`` was incredibly helpful when debugging - and might be in our documentation too?

How about where you add the ``TextIOWrapper`` wrapper, setting ``.url`` at that point for backward compatibility?

Reply:
done